setup.cfg

```diff
@@ -178,7 +178,7 @@ exclude =
     doc/source/getting_started/comparison/includes/*.rst
 
 [codespell]
-ignore-words-list = ba,blocs,coo,hist,nd,sav,ser,bu,ro,recuse,fromm
```

Comment:
@MarcoGorelli created a PR to CodeSpell, but for now should we keep recuse in Pandas ignore list?